---
permalink: /vitro/
title: Labwork

gallery_research:
  - url: /assets/images/labwork1_granules.jpg
    image_path: /assets/images/labwork1_granules.jpg
    alt: "granules"
    title: "Pretreating raw Olburgen wastewater biomass (top) by washing granules (bottom)"
  - url: /assets/images/labwork2_diatoms.jpg 
    image_path: /assets/images/labwork2_diatoms.jpg
    alt: "diatom"
    title: "Pickled diatom, why are you still green?"
  - url: /assets/images/labwork3_enrichment.jpg
    image_path: /assets/images/labwork3_enrichment.jpg
    alt: "Enrichment cultures"
    title: "Anaerobic enrichment cultures"

---

For a thorough study of geobiology (or environmental microbiology) I believe it is vital to combine observations from the field with experiments in the lab.
More recently, a third component can be added to the list: sequence analysis, and bioinformatics more broadly. On the "in Situ", "in Vitro", 
and "in Silico" pages of this site, I highlight some of my research and teaching efforts in these three components of geobiology.     
   
This page focuses on my efforts in "wet" labwork.      
    
    
### Teaching   
   
**GeoBio course molecular biology module**   
The International Geobiology course brings together students from fields adjacent to geobiology (Bio/Chem/Geo) and gives them a taste of "the other side".
As such, the Caltech hosted installments feature a whirlwind of different lab techniques, from nanoSIMS, to petrography, to molecular biology. For the last
three years, I have been responsible for teaching the molecular biology section, which involves DNA extraction, a 16S rRNA gene PCR, and gel 
electrophoresis. The follow-up barcoding PCR and pooling is then done by Dr. Stephanie Connon, and the turnaround time of the amplicon sequencing 
is short enough that the students can then analyse their results in the final weeks of the course. The module starts with a refresher lecture on how DNA
extraction, PCR, and gel electrophoresis work, and why we use the rRNA genes for amplicon sequencing. Using the Powersoil kit, and the highly processive 
Q5 polymerase, the labwork fits in a single day, with the students extracting 6 samples each.   
   
    
**Undergraduate (micro)biology courses**  
As an undergraduate student, MSc student, and PhD student at the Radboud university, I have TA'd the practical components of a wide range of biology
courses, but it is the "introduction to microbiology" that I am the most fond off. While most students are aware microbes exist, the extent of their 
presence in everyday life and their role in society is frequently underappreciated. Guiding students through that eye-opening experience is something
i fondly hope I'll be involved in again soon.    
    
    
### Research   

{% include gallery id="gallery_research" caption="Wet lab research, I clearly do not take enough photos at work..." %}
   
   
**Experimental design for metagenomics studies**   
At its most straightforward, the lab component of metagenomic sequencing involves extracting DNA from an environment, preparing a library for the
sequencing method of choice, and sequencing the library. In many cases, the second and third steps are done through a commercial provider. To facilitate 
downstream analysis and metagenome assembled genome (MAG) recovery, having multiple related samples (e.g. horizons of fhe same sediment core) is very
helpful. Alternatively, or in addition, samples can be treated or partitioned to get meaningful information about the physiology of the constiutuent 
organisms. For example, in our study of a [partial nitritation reactor](https://www.nature.com/articles/ncomms11172) at a wastewater treatment plant,
washing the granules enriched for the organisms inhabitng the anaerobic niches only found in the centers of the granules. Designing sequecning 
experiments with the environmental structure in mind can greatly enhance the interpretation of the microbial ecosystem function.   
     
**Co-cultivation of "unculturable" microorganisms**  
The rapid advances in DNA sequencing and analysis of sequencing data have been transformative for our understanding of microbial diversity. The 
developement of genome resolved metagenomics has added many new branches to the tree of life, but one of the major challenges in the interpretation 
of this wealth of data is the lack of characterized organisms in these new branches. The confidence of our metabolic predictions correlates with the 
evolutionary distance to microorganisms that have been characterized in the lab. Obtaining cultured representatives across the tree of life will be
key for continued growth of our knowledge. To achieve this, we must ask ourselves: why are organisms thought to be "unculturable"?     
One answer to this question could be growth rate, which is hard to tackle. But it is not the only possible answer. Another possibility is lack of 
knowledge on growth requirements of uncultured organisms. One way we can attempt to migitate this, is letting other members of the microbial community
do that work for us. Using knowledge on the structural organization of microbial communities, we can enrich microbes with their natural partners 
in reduced communities. Those reduced communities will be a valuable resource for eludication of the principles behind microbial "unculturability".   
    
**Outlook: using sequence information to prioritize protein characterization**   
As we use the genomic toolbox to explore more and more ecosystems, and add more divergent branches to the tree of life, more protein families will
emerge and existing ones will expand. We primarily annotate all these new proteins using homology, which is incredibly sensitive to error propagation.
So, with increasing numbers of protein sequences available, it will pay to take a hard look at the protein families we know something about and see if it is 
indeed reasonable to assume all members of the family catalyze the same (set of) reactions. Protein classification is a way to approach this challenge,
and I briefly discuss my take on this, [ASM-clust](https://www.biorxiv.org/content/10.1101/792739v1), on the "Silico" page of this site. But classification
is mostly useful as a way to prioritize targets for experimental verification. This is one of the challenges I aim to address in my work going forward.
       
